plugins/woocommerce/i18n/states.php

```diff
@@ -1863,7 +1863,71 @@
 		'Y' => __( 'Delta Amacuro', 'woocommerce' ),
 		'Z' => __( 'Amazonas', 'woocommerce' ),
 	),
-	'VN' => array(),
+	'VN' => array(
```

Comment:
It would be nice if we can cite the source. 
I think we can mention either or both: 

- https://www.gso.gov.vn/phuong-phap-thong-ke/danh-muc/don-vi-hanh-chinh/
- https://en.wikipedia.org/wiki/Provinces_of_Vietnam